magento/magento2
# Issue 23376

### Description (*)

I have fixed the image upload issue for categories. Whenever an image is uploaded with same name , it will be renamed in target/main directory.

### Fixed Issues (if relevant)

1. #23376 : Category A image replaced by uploading to Category B
### Manual testing scenarios (*)

1. Please upload image with name let's say dummy.jpg in  category A
2. Take another image with the same name and upload the image to another category B.
3. Now observe images of both the categories A & B.

### Contribution checklist (*)
